library/sbi-spec/src/binary.rs

```diff
@@ -62,38 +76,54 @@ pub const RET_ERR_IO: usize = <usize as SbiRegister>::RET_ERR_IO;
 /// Implemented automatically for all types that satisfy `Copy`, `Eq`, and `Debug`.
 pub trait SbiRegister: Copy + Eq + Ord + core::fmt::Debug {
     /// SBI success state return value.
+    #[doc(alias = "sbi_ret_success")]
```

Comment:
```suggestion
```

Current Rust langauge does not allow us to mark trait internal constants; also, it is not a function in RISC-V SBI standard.